pkg/ui/app/components/sortabletable.tsx

```diff
@@ -91,37 +92,38 @@ export class SortableTable extends React.Component<TableProps, {}> {
   render() {
     let { sortSetting, columns } = this.props;
 
-    return <table>
+    return <table className="sort-table">
      <thead>
-        <tr className="column">
+        <tr className="sort-table__row sort-table__row--header">
```

Comment:
Well, it's still a row (tr element), it's just the header row.  It looks different than other rows because it is the header row, but still shares common traits with the other rows in the table.